_commands/basics/cd.md

```diff
@@ -16,7 +16,88 @@ $ cd /home
 
 ### Useful Options / Examples
 
+
 #### `cd -`
 
-Jump back directory
+Jump to the last directory that you were in. This will not always necessarily be the parent to your current directory, which would instead be the case if you were to use `cd ..`
+
+(Note that this command prints current path after completion)
+
+~~~ bash
+Alborz-Brals-MacBook-Pro:basics abral$ pwd
+/Users/abral/Desktop/Computer_Science/EECS 398 (C4CS)/adv_HW_7_2/c4cs.github.io/_commands/basics
+Alborz-Brals-MacBook-Pro:basics abral$ cd /tmp
+Alborz-Brals-MacBook-Pro:tmp abral$ pwd
+/tmp
+Alborz-Brals-MacBook-Pro:tmp abral$ cd -
+/Users/abral/Desktop/Computer_Science/EECS 398 (C4CS)/adv_HW_7_2/c4cs.github.io/_commands/basics
+~~~
+
+
+#### `cd ..`
+
+Another way to jump back directory
+
+The two dot characters `..` represent the parent directory to the current directory in which we are working
+
+~~~ bash
+Alborz-Brals-MacBook-Pro:basics abral$ pwd
+/Users/abral/Desktop/Computer_Science/EECS 398 (C4CS)/adv_HW_7/c4cs.github.io/_commands/basics
+Alborz-Brals-MacBook-Pro:basics abral$ cd ..
+Alborz-Brals-MacBook-Pro:_commands abral$ pwd
+/Users/abral/Desktop/Computer_Science/EECS 398 (C4CS)/adv_HW_7/c4cs.github.io/_commands
+~~~
+
+#### `cd .`
+
+The one dot character `.` represents the current directory in which we are working. This command simply moves us to our current directory
+
+~~~ bash
+Alborz-Brals-MacBook-Pro:basics abral$ pwd
+/Users/abral/Desktop/Computer_Science/EECS 398 (C4CS)/adv_HW_7/c4cs.github.io/_commands/basics
+Alborz-Brals-MacBook-Pro:basics abral$ cd .
+Alborz-Brals-MacBook-Pro:basics abral$ pwd
+/Users/abral/Desktop/Computer_Science/EECS 398 (C4CS)/adv_HW_7/c4cs.github.io/_commands/basics
+~~~
+
+Of course, this accomplishes nothing; however, there are other uses for the `.` character pertaining to the `cd` command:
 
+`.` can mean "source", as in the following example
+
+~~~ bash
+./my_script
+~~~ 
+
+However, note that when we use `.`, it runs "my_script" as an executable, and launches a new shell to do so. On the other hand, if we were to use `source` instead, like so:
```

Comment:
Sure, that works! I'll just include a link to that page then, and a brief description of what is on it, if that sounds okay.